### DISCUSSION TITLE
Welcome to dispatch-docker Discussions!

### DISCUSSION THREAD

**Discussion by mvilanova** [Netflix/dispatch-docker#93] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dimuit86** [comment]
Hi Guys,
I'm just curious about how to login to dispatch after the installation. Where do I need to mention username and passwords? To have an email authentication, what should I pass into .env? 

Thanks

**Reply by mvilanova** [reply]
Hi @dimuit86 , you can find instructions on how to set up auth [here](https://hawkins.gitbook.io/dispatch/configuration/app#authentication). Then go to http://localhost:8000/auth/register to register a new user and http://localhost:8000/auth/login to login.
